python/tvm/testing.py

```diff
@@ -714,4 +714,33 @@ def func(f):
     return wrap(args)
 
 
+def compare_numpy_tvm(inputs, output, target, ctx, compute, schedule):
```

Comment:
I wonder if this would better live in `topi.testing` since it depends on TOPI compute and schedule.